sync-gateway-config.json

```diff
@@ -160,12 +161,15 @@ function(doc, oldDoc){
 
     /* Route */
     // Add doc to task-list users and moderators channel.
-    channel("task-list." + doc.taskList.id + ".users");
+    var list = doc._deleted ? oldDoc.taskList.id : doc.taskList.id;
```

Comment:
At this point I don't think you've checked that oldDoc.taskList exists, or the taskList.id fields are non-empty for either doc or oldDoc.